dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -1963,6 +1978,12 @@ class ProviderPRInfo(NamedTuple):
                 )
                 continue
             prs = get_prs_for_package(provider_id)
+            if prs == [-1]:
```

Comment:
Why not just `if prs == []` without any of the changes above?

Reply:
or even (`if not prs`) ?

Reply:
Actually you are right. A classic example of overengineering or KISS principle in play!